ios/RNTrackPlayer/Models/Track.swift

```diff
@@ -124,11 +127,18 @@ class Track: AudioItem, TimePitching, AssetOptionsProviding {
     // MARK: - Authorizing Protocol
 
     func getAssetOptions() -> [String: Any] {
+        var options: [String: Any] = [:]
         if let headers = headers {
-            return ["AVURLAssetHTTPHeaderFieldsKey": headers]
+            options["AVURLAssetHTTPHeaderFieldsKey"] = headers
         }
-
-        return [:]
+        if #available(iOS 16, *) {
```

Comment:
Is there any way to also add the user agent when the ios version is lower than 16?